style.css

```diff
@@ -86,3 +95,160 @@ main {
 .arrow-icon {
   text-align: center;
 }
+
+.flex {
+  display: flex;
+  padding-bottom: 20px;
+  width: 100%;
+}
+
+.column {
+  flex-direction: column;
+}
+
+.works-section-title {
+  text-align: center;
+  font-size: 40px;
+  color: #172b4d;
+}
+
+.works-grid-container {
+  display: grid;
+  gap: 24px;
+  justify-content: center;
+  width: 86%;
+  margin: 62px auto;
+}
+
+.card img {
+  width: 327px;
+  max-width: 100%;
+  height: 252px;
+}
+
+.card {
+  width: 100%;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+button {
+  cursor: pointer;
+}
+
+.card-details {
+  align-items: center;
+  gap: 12px;
+  color: #172b4d;
+}
+
+.card-details h2 {
+  margin-top: 16px;
+}
+
+.card-details ul {
+  margin-right: 20px;
+}
+
+.list {
+  gap: 8px;
+}
+
+.card-details button {
+  margin: 15px 102px 24px 103px;
+  padding: 12px;
+  background-color: #36b37f;
+  border: none;
+  border-radius: 4px;
+  color: #fff;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.about-me {
+  background-image: url(./Icons/about-top.svg), url(./Icons/about_bg.svg);
+  background-repeat: no-repeat;
+  background-position: right -40px, 0 270px;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.about-me button {
+  margin: 15px 102px 24px 103px;
+  padding: 8px;
+  background-color: #36b37f;
+  border: none;
+  border-radius: 4px;
+  color: #fff;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+button:hover {
+  background: #36b37f;
+  box-shadow: 0 8px 16px rgba(54, 179, 127, 0.24);
+}
+
+button:active {
+  background: #008552;
+}
+
+.about-me h2 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.about-me p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  margin: 20px 15px 0 15px;
+  color: #42526e;
+}
+
+.about-me-grid {
+  display: grid;
+  gap: 24px;
+  justify-content: center;
+  width: 100%;
+  margin: 62px auto;
+}
+
+.about-me-grid-item {
+  background: #ebf0ee;
+  border: 1px solid #c1c7d0;
+  border-radius: 8px;
+  align-items: center;
+  justify-content: center;
+  gap: 24px;
+  text-align: center;
+  height: 316px;
+}
+
+.about-me-grid-item h3 {
+  color: #172b4d;
+}
+
+.about-me-grid-item ul {
+  color: #36b37e;
+}
+
+.iconic {
+  background-color: #fff;
+  padding: 18px;
+  border-radius: 50%;
+  margin-top: 40px;
+  box-shadow: 0 16px 24px rgba(37, 47, 137, 0.08);
+}
+
+.skills {
+  flex-wrap: wrap;
+  justify-content: center;
+  max-width: 285px;
+}
```

Comment:
- [ ] I can see some extra dots on the left side of the list of skill items like `. Ruby on Rails, `. React.js`, etc. Kindly remove them.  You use list-style to fix this and consider reading the [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/list-style) for how to do this.
- [ ] Kindly use the given font for every `hx`, `p` and text items. You can browse the given template and see which font is needed for which section.

- Your Design
- ![image](https://user-images.githubusercontent.com/60083437/227739513-14ab7a05-e05b-4827-9850-9e87fa5352c4.png)

Reply:
- Hey, as requested by the previous reviewer, the list items should not have those dots. you can remove them by removing the list style.

``` 
ul,
li {
  list-style-type: none;
}
```